### DISCUSSION TITLE
Demo: Lume 3D HTML Elements (using JSPM importmap)

### DISCUSSION THREAD

**Discussion by trusktr** [jspm/project#309] [Show and tell]
[JSPM Sandbox Link](https://jspm.org/sandbox#H4sIAAAAAAAAA+VWUW/bNhB+rn/FVcWAZLUs2V4y17aCpE1QtGiSrm7QDUMfaIm22FCiQNJxlKL/fUdScmKnVjKsDxsWRJZ4PH733fHIu/FT34fXJ2cnH44+nn+AV+fHJxD80z/w/YPW+GkiYl0WFFKdcRzXL0qSgxbAOKOaQJwSqaiOvIWe+QPPTmimOT24yO07GQduvFqSk4xG3hWjy0JI7UEsck1zhFiyRKdRQq9YTH07aAPLmWaE+yomnEZdNDAOHIPxVCSlRcUQ4Avg7eT9KbymOZVECwlvMoMPp6SwsycJ03Dx4d0Q/dGFGgbBvFbtfFFF1mEieHbRO0v/eP3y8vg0lOXk5u3kopuWr/Kzj79fLMPj959OX7w8u3mVv5mc/Xaj5kdHz/uXg3P25ggtmJihmbGKJSs0mMhFHrMcMlLYyHy1RCqh8oaVAEV8kVEceytqZMUpL7KhmT4MO/1O6BNepKTTGwQJUzpgeUKvUdWzQN/azoCKRUHX8L+De2caFQ6NiYAstOCkFAvdQGZTFYl18d8ROkLxOys2rNqbBuKF0iLzidaSTReaqg03HrJ6bz0a79W2Vyj3zFJOM0yxB+ErvTWPGlCvUHe24A/DVopuCx+CvWRL9iCkUUK4X2qWZvxdNJ1KSv1Cii801jTxM6KpxBPllr2v5aeV+DGbsA3SutdtAL5H7org1JQ3pf664oM7U+2gP6UpuWJCNnlzT/dwr9MI/gVvLuVPRZ40oN4qIdlfm7abi2XMiWqiWKsc7iOzBihJ5/RaNgA5hcMewvS2wyjBWeI3pkCtctjF5KtuISvbAhWYsvF4PKPtQM3XNky8BGRT0myAWvWKqvncBruk08eDorKDxI8NQHtAGpDsvEnlPZMf0wXjiTtUnUwkC77Jz04Fj8bzqpXfWvWvecaBK0wHrapkwskETq05mKQsU8O6YGK1UlAIXs4Y5zDDSip4QiVMpVgqKhUsmU7xggd2R18tCjvYoXOYkBkeV+jud/q7VVmsqyJRZR6DknG09VAq3wXBV4YVBntQJ/7GlIkSxFIoJSSbszzySC7yMhML5R2su7tWkx2GLchQ+/AzMgNz08BMiqyqx6OWVcF+6JNkmqKjLAe3GpuWBBsjKmkbMEC4mICiBcF2AiEYKpggodBDXz1Y1SsQOeiU1iiOV8eawTYIM4x2uJjvGPO7o7VdGweu3xkHrg9r2T08Oj4+Of6hXZ/SpWvWjJk2GKNVn5ARiVEeQjiyw5SyeaqH0A3Dn0atJ0+mJL6cS7HIkyE8G/T7AxqPbO6hDw6zNTZ+YSuHfRfgqZlzKNJSMWzteOnHQkqsGD43sJiAmGkFjtkVbtkgDD3MxLnZfxxyllMinSQWXMjIqyw6mZmNvP160cyMXuDIZqLlEOMlLYmPeQMEDeZ+IZhpQcPOHuDjrfpOk3gkjysOmFYby11+WaGFcPRxOba0iuky8ro9w+P7VgqBOkxg6vbDEHzz0zfatVOzWRju16laRa8wWQeK3SCnntElSuOBIIlYRt6McEU9wEBSDNymeAMWUqIib0pwB1Y1fOWis2MjFmy696Ocrh22nt/S65PB/mz2w72uYP89Xv+fnF7lt/8fS/CpuAaWIIq4/ltUrOn6eru1DnhDztOcKmXCNNgWOCxiyGVd6mbquIRQPSuvlykWqbsxDwHDvXfn2kIfTAG5vYXNnbytNqJyRwpNDBZEsHPdhnIXogP48/o5MsGReX0eVaq12Vq1DTeVth1g99rDKruDjMzEc7j5PDKVlVNZtFZV7i/XAUXLzhAAAA==)

![](https://github.com/jspm/project/assets/297678/011131cb-2ee8-4838-9933-162a035ef90c)

**Comment by guybedford** [comment]
Great to see you got things working :) Always happy to work through issues, so thanks for posting.
